infra/charts/turing/templates/_helpers.tpl

```diff
@@ -202,6 +202,7 @@ RouterDefaults:
 {{ if eq (toString $expEngine.type) "rpc-plugin" }}
       PluginConfig:
         Image: {{ $expEngine.rpcPlugin.image }}
+        LivenessPeriodSeconds: {{ $expEngine.rpcPlugin.livenessPeriodSeconds }}
```

Comment:
Shall we set a default value here so that the Turing operator can potentially skip setting this value during deployment (`| default 10`) ?